apis/Google.Logging.V2/Google.Logging.Log4Net/Extensions.cs

```diff
@@ -29,12 +29,16 @@ internal static class Extensions
 
```

Comment:
While I'm here, the Timestamp method:

- Looks odd given that it means there's both a property and a method called Timestamp, unless I'm missing something
- Can use the ToDateTime method of Timestamp

Additionally, what's the Scale method meant to be for?